spyder/plugins/editor/plugin.py

```diff
@@ -377,6 +377,14 @@ def closing_plugin(self, cancelable=False):
             else:
                 for win in self.editorwindows[:]:
                     win.close()
+                # Remove autosave file on successful close to avoid #9265

```

Comment:
... to avoid *issue* 9265 ...

Reply:
This is our convention to reference issues.

Reply:
Done